test/name.spec.js

```diff
@@ -99,7 +99,7 @@ describe('.name', () => {
         .then((res) => {
           expect(res).to.exist
           expect(res).to.be.eql({
-            Path: '/ipfs/' + name.Value
+            Path: name.Value
```

Comment:
This sounds like a breaking change in go-ipfs land. @whyrusleeping why has go-ipfs stopped returning the resolved hash with /ipfs/?